tools/.github-cla-signers

```diff
@@ -28,4 +28,6 @@ sshedi
 TheRealFalcon
 tomponline
 tsanghan
+vijarad1
```

Comment:
I confirm that internal Canonical CLI documentation validates @vijarad1 has signed the CLA.